x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/WaterfallItem.tsx

```diff
@@ -10,6 +10,7 @@ import styled from 'styled-components';
 import { EuiIcon, EuiText, EuiTitle } from '@elastic/eui';
 import theme from '@elastic/eui/dist/eui_theme_light.json';
 import { asTime } from 'x-pack/plugins/apm/public/utils/formatters';
+import { isRumAgentName } from '../../../../../../../server/lib/apm_telemetry/agent_name';
```

Comment:
the source file doesn't have any dependencies, so it's safe to import, but we can move it to another location if it doesn't make sense in `server/lib`. The reason i didn't add it to `typings` is because it also has runtime code.

Reply:
Yeah, it should probably be moved to `common`.